internal/controller/operatorconfigmap_controller.go

```diff
@@ -801,6 +811,14 @@ func (c *OperatorConfigMapReconciler) getDeployCSIConfig() (bool, error) {
 	return deployCSI, nil
 }
 
+func (c *OperatorConfigMapReconciler) getNoobaaSubManagementConfig() bool {
+	manageNoobaaSubAsString := cmp.Or(
+		c.operatorConfigMap.Data[manageNoobaaSubKey],
+		strconv.FormatBool(true),
+	)
+	return strings.ToLower(manageNoobaaSubAsString) != strconv.FormatBool(false)
```

Comment:
1. There is value in loging an error in case user provided a value in to config map but it does not match an expected value.
2. You can eliminate a lof of the conversions here between string and bools

```suggestion
	valAsString, ok := manageNoobaaSubAsString[manageNoobaaSubKey]
     	if !ok {
        	return true
     	}

	if val, err := strconv.ParseBool(valAsString); err != nil {
		c.Log.Error(
			err, 
			"Unsupported value under manageNoobaaSubscription key", 
		)
		return true
	} else {
		return val
	}
```